tgui/packages/tgui/components/TooltipOverflow.js

```diff
@@ -0,0 +1,63 @@
+import React from 'common/react';
```

Comment:
no license identifier? should have style's iirc

Reply:
File removed